enaml/core/import_hooks.py

```diff
@@ -337,7 +339,27 @@ def _get_magic_info(self, file_info):
             timestamp = struct.unpack('i', cache_file.read(4))[0]
         return (magic, timestamp)
 
-    def compile_code(self):
+    def read_source(self, src_path):
```

Comment:
I do not see the point of passing explicitly the src_path, everything is in the file_info so I would simply use that.

Reply:
See further comments for how to avoid code duplication in the ZipImporter